proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -18,7 +18,7 @@ message BaseAccount {
   option (cosmos_proto.implements_interface) = "AccountI";
 
   string              address        = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  google.protobuf.Any pub_key        = 2 [(gogoproto.jsontag) = "public_key,omitempty"];
+  google.protobuf.Any pub_key        = 2;
```

Comment:
@marbar3778 could we add a changelog entry for client-breaking with all the breaking changes 🙏  e.g. this one `public_key -> pub_key`

Reply:
yes, I will add everything before marking it as ready

Reply:
we could still keep backwrds compatibility by adding the protobuf `json_name` option to fields

Reply:
also IIRC, protojson is relaxed on how it handles snake case vs camel case (note protojson defaults to camel case iirc). if clients are processing requests using protojson then all should be fine besides some special cases such as pub_key.

Reply:
perfect this is what I was unsure on. Ill replace gogoproto.jsontag with json_tag